modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -46,36 +46,98 @@ Some actions are not yet supported:
 
 This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
 
-////
-.Procedure: Preparing an {Ubuntu} 16.04 Client for Registration
+// SUSE Manager specific instructions
+//ifeval::[{suma-content} == true]
 
+.Procedure: Setting up the software channels for {ubuntu} Client registration
+
+Ensure you have the corresponding {ubuntu} product enabled and required channels have been fully synchronized:
+
+.Ubuntu 18.04
+* Product: {ubuntu} Client 18.04
+* Mandatory channels: [systemitem]``ubuntu-18.04-pool-amd64``
++
+.Ubuntu 16.04
+* Product: {ubuntu} Client 16.04
+* Mandatory channels: [systemitem]``ubuntu-16.04-pool-amd64``
+
+
+After synchronization the following vendor channels will be created:
+
+----
+ ubuntu-18.04-pool for amd64
+ |
+ +- Ubuntu-18.04-SUSE-Manager-Tools for amd64
+----
+
+[NOTE]
+====
+The {ubuntu} vendor channels don't contain {ubuntu} upstream packages.
+The repositories and channels for syncing upstream content must be configured manually.
+====
+
+Next create custom repositories to mirror the {ubuntu} packages (see <<manage-repositories.adoc>>):
+
+For e.g. create repositories for the `main` and `main-updates` components:
+
+* Repository Label: ubuntu-bionic-main
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/
+* Repository Type: deb
+
+* Repository Label: ubuntu-bionic-main-updates
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/
+* Repository Type: deb
+
+Create custom channels under the `pool` channel and associate them with the repositories created in the previous step.
+
```

Comment:
Should we add a section about how to create the "Channel"? Important there is to set the correct architecture which I miss here.